webapp/components/survey/NodeDefDetails/ExpressionsProp/NodeDefExpressionsProp.js

```diff
@@ -24,12 +24,12 @@ const NodeDefExpressionsProp = (props) => {
     readOnly,
     isContextParent,
     canBeConstant,
-    isBoolean,
     hideAdvanced,
     mode,
   } = props
 
   const nodeDef = State.getNodeDef(state)
+  const isBoolean = NodeDef.isBoolean(nodeDef)
```

Comment:
why this change ?
this refers to expression type, not nodeDef.
you might break the behavior of data all boolean expressions used